src/validation.cpp

```diff
@@ -3802,9 +3815,9 @@ static FILE* OpenUndoFile(const CDiskBlockPos &pos, bool fReadOnly) {
     return OpenDiskFile(pos, "rev", fReadOnly);
 }
 
-fs::path GetBlockPosFilename(const CDiskBlockPos &pos, const char *prefix)
```

Comment:
Commit 88cf780

nit, remove?

Reply:
It's used once in `init.cpp`.

Reply:
That would mean require `BlockFileSeq`, which is currently static and a broader interface.

Reply:
Right, just realized it's a static function in `validation.cpp`